### PR TITLE
Added aha scanner

### PR DESCRIPTION
**Pull request by roxanne-tampus**

### CODE REVIEW

File: pkg/detectors/aha/aha.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"aha"}) + `\b([0-9a-zA-Z]{64})\b`)
```
Comment (ahrav) [inline pkg/detectors/aha/aha.go:24]: suggestion: You can tighten up this regex pattern by using `\b([0-9a-f]{64})\b` Aha's API keys are encoded so they only use a-f0-9.

**Review by ahrav — APPROVED** [review]
small suggestion otherwise lgtm.

File: pkg/detectors/aha/aha.go (hunk 1 above)
Comment (roxanne-tampus) [reply pkg/detectors/aha/aha.go]: I just modified the regex as suggested. Thanks @ahrav!

Comment (dustin-decker) [reply pkg/detectors/aha/aha.go]: Some extra context: it's hex encoding that uses the a-f characters rather than a-z

**Pull request merged**